gvr-vuforia/src/com/qualcomm/vuforia/misc/VuforiaApplicationSession.java

```diff
@@ -30,7 +30,8 @@
 {
     
     private static final String LOGTAG = "Vuforia_Sample_Applications";
-    
+
+    private final String licenseKey = "AVUdgen/////AAAAARstQwK5r0TMhV1r5rVYLfoK6mEkcm+9dxNOSuZscys6w4uSsJyAFl/hbJohWnMF1grq0p2GqHZrWDh4TqP6x37RFm7p1zxQgdlDWZz8R/78Ga8zLdah23OXohLjlZ1/Y7+pHLqAyF2KQKIoH2HH9Wz47vVKidGQRY9R25QzV0cJKCvt6Zmjfryv5yrcsI20ZQVp7A3dX1G1MCSOwAWamtCN59LYHuRAr01xjK3eZWKquzQzJKmTxcczzl322+IxyGknlY63QVvS7DZrPS8dRS/tRVgyQYGqPSd/cfDWInDUC3k1SynzkvoXJKkZ9JPsCaFBCgiyLg3d/y0lsgT6AoOkRjuY4EQEyVT+lDC0mRBc";
```

Comment:
how is this license key generated?  how long is it good for?  will it work on anyone's phone?